src/lib/settings/schemas/UserSchema.ts

```diff
@@ -60,4 +64,35 @@ Client.defaultUserSchema
 			.add('range', 'any', { default: Gear.defaultGear })
 			.add('misc', 'any', { default: Gear.defaultGear })
 			.add('skilling', 'any', { default: Gear.defaultGear })
+	)
+	.add('farmingpatches', (folder: SchemaFolder) =>
+		folder
+			.add('herb', 'any', { default: Farming.defaultPatches })
+			.add('fruittree', 'any', { default: Farming.defaultPatches })
+			.add('tree', 'any', { default: Farming.defaultPatches })
+			.add('allotment', 'any', { default: Farming.defaultPatches })
+			.add('cactus', 'any', { default: Farming.defaultPatches })
+			.add('bush', 'any', { default: Farming.defaultPatches })
+			.add('spirit', 'any', { default: Farming.defaultPatches })
+			.add('hardwood', 'any', { default: Farming.defaultPatches })
+			.add('seaweed', 'any', { default: Farming.defaultPatches })
+			.add('vine', 'any', { default: Farming.defaultPatches })
+			.add('calquat', 'any', { default: Farming.defaultPatches })
+			.add('redwood', 'any', { default: Farming.defaultPatches })
+			.add('crystal', 'any', { default: Farming.defaultPatches })
+			.add('celastrus', 'any', { default: Farming.defaultPatches })
+			.add('hespori', 'any', { default: Farming.defaultPatches })
+			.add('flower', 'any', { default: Farming.defaultPatches })
```

Comment:
it would be highly preferable for these to use an enum, like i do for SkillsEnum above.